app/models/concerns/media_crowdtangle_item.rb

```diff
@@ -17,6 +17,7 @@ def self.crowdtangle_request(resource, id)
 
         begin
           response = http.request(request)
+          Rails.logger.info level: 'INFO', message: '[Parser] Requesting data from Crowdtangle', url: uri.to_s
```

Comment:
Is `level: 'INFO'` needed when you're calling `Rails.logger.info`?

Reply:
It is not necessary, I only have it here for uniformity since all other occurrences in `pender` have it like this. We can remove it.

Reply:
Thanks, fine to keep it then.

Reply:
Thank you!

Could you please hit 'Merge' for me?  I'm still waiting for my permissions to be sorted.

Reply:
Sure, done!